### DISCUSSION TITLE
Emoji Saver plugin maybe?

### DISCUSSION THREAD

**Discussion by Syntruss** [bepvte/bd-addons#15] [Ideas]
i have a lot of favorited emojis. every time discord crashes, I have to remember what they are and save them again. is there a way to make a plugin that remembers and keeps them when you crash? something like gif saver does? i think it would be very helpful. 

-syn

**Comment by bepvte** [comment]
Yes, I can do the same with recents and stickers too. I'll add these features to settings soon

**Reply by Syntruss** [reply]
alright cool, thanks
